paramak/parametric_components/blanket_fp.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from scipy.interpolate import interp1d
 import sympy as sp
+import mpmath
```

Comment:
Interesting package this mpmath. Perhaps we should use it more across the rest of the Paramak

If we need to use it here then we should add it to the setup.py and requirments.txt

Reply:
I didn't have to install it so it must be installed by one of the librairies (maybe numpy or sympy). Shall we add it to requirements nonetheless ?

Reply:
> Interesting package this mpmath. Perhaps we should use it more across the rest of the Paramak

I'm using mpmath here instead of math because I need to convert sympy objects. If it's only float, the regular math package can be used (included in python). It it's numpy arrays, then numpy has a `np.radians()` function

Reply:
ah yes, and the CI appears to be happy with mpmath as well